spec/amp-versioning-policy.md

```diff
@@ -0,0 +1,40 @@
+# AMP versioning policy
+
+## Basics
+
+- We use [semver](http://semver.org/) terminology, but see below for changes from simple semver.
+- The AMP JS library is distributed via its major version only. The major version is expected to change rarely if ever (on the order of less than once per year).
+- Each AMP extension is versioned independently. Publishers of AMP documents can choose the major and minor version of an extension. They cannot choose the patch version.
```

Comment:
I don't think the validator has a role in this. This just says how the versions themselves are composed.